### PR TITLE
Update wallet distributor frontend

### PR DESCRIPTION
**Pull request by toshiakitakase**
- Fix API requests(start-execution and describe-execution) fired when a page is loaded without payload
- Show error when invalid json format file is uploaded
- Fix storybook error of Step1 CreateWalletCampaignPageTemplate has error

### CODE REVIEW

File: frontend/packages/react-app/src/components/organisms/WalletDistributionTargets/index.tsx [hunk 1]
```diff
@@ -56,7 +56,7 @@ const WalletDistributionTargets: React.FC<TargetsProps> = ({
                 <Typography variant={"body1"}>Format: </Typography>
                 <Box ml={2}>
                   <Typography variant={"body1"} color={"secondary"}>
-                    {`{"targets": ["walletaddress1", "walletaddress2"], "type": "address"}`}
+                    {`{"targets": ["walletaddress1", "walletaddress2"]}`}
```
Comment (wildmouse) [inline frontend/packages/react-app/src/components/organisms/WalletDistributionTargets/index.tsx:59]: A logic that checks type field is contained in API, so it should be still displayed. 
https://github.com/iroiro/iroiro/blob/1aaa3166a37443a1f423be63835ae296e57aef59/functions/merkle-tree/functions/input-generator/src/app.ts#L35-L37

**Comment by wildmouse** [comment]
When a campaign creation transaction is included in a block, the following error occurred. 
> TypeError: Cannot read property 'endDate' of null

**Comment by toshiakitakase** [comment]
Fixed

**Review by wildmouse — APPROVED** [review]
LGTM 🏃‍♀️

**Pull request merged**